llm-training-course/colab/finetuning.ipynb

```diff
@@ -0,0 +1,1100 @@
+{
```

Comment:
<p>Move title to above the installs,</p><p> </p><p>Link to W&amp;B  HF documentation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/edu/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>updated</p>